Containerfile

```diff
@@ -1,54 +1,38 @@
 FROM ubuntu
 
-LABEL maintainer="vchalla@redhat.com"
-
+# setup installation directory and disable prompts during installation
 WORKDIR /tmp
 ARG DEBIAN_FRONTEND=noninteractive
 
-# ALPINE
+# Install necessary libraries for subsequent commands
 RUN apt-get update
 RUN apt-get upgrade  -y
-RUN apt-get install -y podman
-RUN apt-get install -y wget python3.6 git dumb-init
-
-# Snafu Build Dependencies
-# TODO: A lot of the *-dev dependencies were added while attempting to build
-#       snafu's requirements (mainly numpy, scipy, pandas). Now that these are
-#       installed using Alpine's package manager, they may not all be necessary
-#       anymore.
-RUN apt-get install -y python3-numpy python3-scipy python3-pandas gcc python3-dev \
-                       postgresql-client libffi-dev libxml2 \
-                       libxml2-dev libxslt-dev libjpeg-dev \
-                       zlib1g-dev musl-dev
+RUN apt-get install -y podman wget git dumb-init python3.6 python3-distutils python3-pip python3-apt
```

Comment:
Don't `update` and `upgrade` -- you should rely on the image you're building on being up to date.

All the other `apt-get` commands need to happen in one step -- running `apt-get clean` only makes the container image larger.

Reply:
I am sorry, we will have to run update in order for all the dependent python packages to get installed. And also it is stated as one of the best practices: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run